src/SslCertificate.php

```diff
@@ -188,7 +188,7 @@ protected function wildcardHostCoversHost(string $wildcardHost, string $host): b
 
         $wildcardHostWithoutWildcard = substr($wildcardHost, 2);
 
-        return substr_count($wildcardHost, '.') >= substr_count($host, '.') && ends_with($host, $wildcardHostWithoutWildcard);
+        return substr_count($wildcardHost, '.') >= substr_count($host, '.') && ends_with('.'.$host, '.'.$wildcardHostWithoutWildcard);
```

Comment:
Generally speaking I don't like long `if` statements with `&&`

Could you break this up in multiple `if` statements and early returns?

Reply:
done